### PR TITLE
docs(validation): add backend validator example

### PR DESCRIPTION
**Pull request by MathieuPuech**
## What I did

1. Add a backend validator example

### CODE REVIEW

File: packages/form-integrations/docs/17-validation-examples.md [hunk 1]
```diff
@@ -695,3 +695,80 @@ export const FormValidationReset = () => html`
...
+  return html`
+    <lion-form @submit=${submitHandler}>
+      <form>
+        <lion-input
+          label="username"
+          name="username"
+          .validators="${[new BackendValidator(''), new Required()]}"
+          .modelValue="${''}"
+        ></lion-input>
+        <lion-checkbox-group .mulipleChoice="${false}" name="simulateError">
```
Comment (jorenbroekema) [inline packages/form-integrations/docs/17-validation-examples.md:766]:
```suggestion
        <lion-checkbox-group .multipleChoice="${false}" name="simulateError">
```

**Review by jorenbroekema — APPROVED** [review]
I like this approach! nice. Comment is just a small nitpick, perhaps multipleChoice prop binding can be left out entirely?

**Comment by MathieuPuech** [comment]
> I like this approach! nice. Comment is just a small nitpick, perhaps multipleChoice prop binding can be left out entirely?

yes, I removed it completely and added some improvements

File: packages/form-integrations/docs/17-validation-examples.md [hunk 2]
```diff
@@ -695,3 +695,92 @@ export const FormValidationReset = () => html`
...
+      }
+      return false;
+    }
+    static getMessage({ fieldName, modelValue, params: param }) {
+      return backendErrorMessage;
+    }
+  }
+  return html`
+    <style>
+      lion-input[is-pending] {
```
Comment (jorenbroekema) [inline packages/form-integrations/docs/17-validation-examples.md:766]: Where does is-pending attribute come from?

Comment (MathieuPuech) [reply packages/form-integrations/docs/17-validation-examples.md:766]: It's added by the `async` validators

https://github.com/ing-bank/lion/blob/ee85fa036e3a7f79e93ea391fec181657833eadb/packages/form-core/src/validate/ValidateMixin.js#L71

Comment (jorenbroekema) [reply packages/form-integrations/docs/17-validation-examples.md:766]: Oooh cool, didn't know!

**Comment by jorenbroekema** [comment]
Looks good to me! Feel free to merge whenever

**Pull request merged**